package.json

```diff
@@ -55,26 +56,26 @@
     "eslint": "^5.16.0",
     "faucet": "^0.0.1",
     "jsonpath": "^1.1.1",
+    "mkdirp": "^0.5.6",
+    "node-addon-api": "^5.0.0",
     "node-timeout": "0.0.4",
     "polyline": "^0.2.0",
     "request": "^2.88.2",
+    "rimraf": "^2.7.1",
     "tape": "^4.16.0",
     "turf": "^3.0.14",
     "uglify-js": "^3.17.0",
     "xmlbuilder": "^4.2.1",
-    "cheap-ruler": "^3.0.2",
-    "mkdirp": "^0.5.6",
-    "nan": "^2.17.0",
-    "node-cmake": "^2.5.1",
-    "rimraf": "^2.7.1"
+    "node-cmake": "^2.5.1"
   },
   "main": "lib/index.js",
   "binary": {
+    "napi_versions": [8],
     "module_name": "node_osrm",
-    "module_path": "./lib/binding/",
+    "module_path": "./lib/binding_napi_v{napi_build_version}/",
```

Comment:
node-pre-gyp is designed to support multiple NAPI versions, that's why they require NAPI version to be part of module_path(as I understand it may be needed if you want to use some features from new versions of NAPI, but still provide builds for older Node versions which do not support this new version of NAPI) https://github.com/mapbox/node-pre-gyp#the-napi_versions-array-property